benchmarks/operator_benchmark/pt/hardsigmoid_test.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import absolute_import
```

Comment:
BTW, for the benchmarks there is some extra work that you need to go through from the FB side